src/styles/_catalog.scss

```diff
@@ -1,22 +1,32 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
+  display: grid;
+  justify-content: center;
+  gap: $card-spacing;
+  max-width: $card-width * 4 + $card-spacing * 3;
   margin: 0 auto;
 
+  @media (max-width: 488px) {
+    grid-template-columns: $card-width;
+  }
+
+  @media (min-width: 488px) and (max-width: 768px) {
+    grid-template-columns: repeat(2, $card-width);
+  }
+
+  @media (min-width: 768px) and (max-width: 1024px) {
+    grid-template-columns: repeat(3, $card-width);
+  }
```

Comment:
No need to specify both min and max width, just keep right order of media queries